### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by bielikb**
* VCs now use translucent navigation bar (except MapViewController & Privacy VC)
* Choosing empty location is not valid, alert will be fired
* Privacy Policy text follows the app font
* About App text updated to the latest version
* custom buttons were converted to system buttons == consistency & native feel

### CODE REVIEW

File: Covid/UI/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import Foundation
+import UIKit
+
+class ViewController: UIViewController {
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        navigationController?.navigationBar.setBackgroundImage(UIImage(), for: UIBarMetrics.default)
+        navigationController?.navigationBar.shadowImage = UIImage()
+        navigationController?.navigationBar.isTranslucent = true
```
Comment (bielikb) [inline Covid/UI/ViewController.swift]: Toto moze byt kludne sucastou napr. UIAppearance, ale zatial som to tymto smerom nepushoval

File: Covid/Base.lproj/Main.storyboard [hunk 2]
```diff
@@ -1471,12 +852,12 @@ ZostanZdravy a Sygic</string>
...
+                            <button opaque="NO" contentMode="scaleToFill" verticalHuggingPriority="252" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="wordWrap" translatesAutoresizingMaskIntoConstraints="NO" id="Njg-l9-GFw">
                                 <rect key="frame" x="24" y="586" width="327" height="66"/>
                                 <color key="backgroundColor" red="0.31372549020000001" green="0.34509803919999998" blue="0.97647058819999999" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
                                 <fontDescription key="fontDescription" name="Poppins-Bold" family="Poppins" pointSize="15"/>
                                 <inset key="contentEdgeInsets" minX="0.0" minY="12" maxX="0.0" maxY="12"/>
-                                <state key="normal" title="Bol som v zahraničí alebo
musím zostať v karanténe">
+                                <state key="normal" title="Bola/bol som v zahraničí alebo
musím zostať v karanténe">
```
Comment (bielikb) [inline Covid/Base.lproj/Main.storyboard:860]: v11 #2

File: Covid/Base.lproj/Main.storyboard [hunk 3]
```diff
@@ -2383,107 +1767,283 @@ nie je COVID-19.</string>
...
-
-Veľa z nás nákazu zvládne svojou imunitou. Sú ale takí, ktorí majú iné vážne ochorenia, či mladí, či starí a budú potrebovať pomoc zdravotníctva. 
-
-Preto je nesmierne dôležité prejaviť ľudskosť mladých voči starým, zdravých voči chorým a poraziť Covid-19, spoločne.
-
+                                        <string key="content">Táto aplikácia bola vytvorená slovenskými odborníkmi-dobrovoľníkmi v rekordne krátkom čase, s cieľom pomôcť v boji proti šíreniu pandémie COVID-19. Bola odovzdaná všetkým krajinám vrátane Slovenskej republiky zdarma, aby pomohla ich zdravotníckym organizáciám urýchliť zvládnutie tejto ničivej nákazy.
```
Comment (bielikb) [inline Covid/Base.lproj/Main.storyboard:1770]: v11 #8

File: Covid/UI/ViewController.swift [hunk 4]
```diff
@@ -0,0 +1,20 @@
+//
```
Comment (anonymcek) [inline Covid/UI/ViewController.swift:23]: licka

File: Covid/UI/ViewController.swift [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+//  Covid
+//
+//  Created by Boris Bielik on 20/04/2020.
+//  Copyright © 2020 Sygic. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class ViewController: UIViewController {
```
Comment (anonymcek) [inline Covid/UI/ViewController.swift:34]: nejaky `TransparentNavBarViewController`, takto to nieje uplne jasne

**Review by anonymcek — CHANGES_REQUESTED** [review]

**Review by anonymcek — APPROVED** [review]

**Pull request merged**